src/App/Components/Header.js

```diff
@@ -1,104 +1,159 @@
 import React from 'react';
 import SideNav from './SideNav';
 import logo from '../../Assets/images/eagle-logo.svg';
-import { faUser } from '@fortawesome/free-solid-svg-icons';
+import { faUser, faBell } from '@fortawesome/free-solid-svg-icons';
 import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
 import Token from '../../helper/helper';
 import LogoutButton from './logout.component';
+import NotificationList from './notification.list';
+import { connect } from 'react-redux';
 
 class Header extends React.Component {
 	state = {
 		show: false,
+		showNotification: false,
 	};
 	handleClick = () => {
 		const currentState = this.state.show;
 		this.setState({ show: !currentState });
 	};
+	handleClickNotification = () => {
+		const currentState = this.state.showNotification;
+		this.setState({ showNotification: !currentState });
+	};
+	getUnreadNotifications(notification) {
+		if (notification) {
+			return notification.filter(nt => nt.isRead === false).length;
+		}
+		return 0;
+	}
 	render() {
 		const validToken = Token();
-		const { showSideNav } = this.props;
+		const { showSideNav, notifications } = this.props;
+		const showNotification = this.state.showNotification ? 'visible' : 'invisible';
 		return (
-			<nav className='navbar navbar-expand-lg bg-light navbar-light fixed-top shadow-sm'>
-				<a className='navbar-brand text-primary' href='/'>
-					<div>
-						<img className='logo' src={logo} alt='logo' />
-						<span className='font-weight-bold'>Barefoot</span>
+			<div>
+				{showSideNav && <NotificationList visibility={`${showNotification}`} />}
+				<nav className='navbar navbar-expand-lg bg-light navbar-light fixed-top'>
```

Comment:
I suggest `ring bell` to be in all pages likes `home`, `accommodations` as long as user is logged in!